src/app/components/nested-data-picker/data/nested-value.interface.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * @author chirila-ioan-daniel
+ */
+export interface NestedValueInterface
+{
+    id:number;
+    isActive:boolean;
+    isOpen:boolean;
+    isVisible:boolean;
+    name:string;
+    tooltip:string;
+    tooltipPlacement:string;
```

Comment:
you can consider to use `TerraPlacementEnum` as typedef

Reply:
ok but in TerraNodeInterface tooltipPlacement is string what should I do?

Reply:
your right. because of the deadline pressure you can ignore it. it will be nice to have but not mandatory